static/app/components/replays/preferences/replayPreferences.tsx

```diff
@@ -5,16 +5,19 @@ const LOCAL_STORAGE_KEY = 'replay-config';
 export type ReplayPrefs = {
   isSkippingInactive: boolean;
   playbackSpeed: number;
+  timestamp: 'relative' | 'absolute';
```

Comment:
nit: maybe this should be named `timestampType` because that's how it's used in the components `timestampType === 'absolute' ? ... : ...`